### ISSUE TITLE
display of citedRange with @unit="entry"

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/project-documentation#84]
@ajaniak: 
Dan is right in a small remark he has made in #66. Please make this change in display:

-  `<citedRange unit="entry">yatan</citedRange>` should display "s.v. _yatan_", not just as "yatan"

or would it be better to encode `<citedRange unit="entry"><foreign>yatan</foreign></citedRange>` explicitly, if I want italics, leaving open the possibility of not having italics in display when they are not wanted?

**Comment by danbalogh** [comment]
I think explicitly encoding italics in such a case is overkill. I understand that some "entry" items may be proper names that we may not want to italicise, and others may be common terms that we would want to - but is this distinction worth the bother? I would be happy with the contents of the entry in italics or regular type, regardless of what kind of word they are.
I could certainly live with either
A. "s.v. _yatan_" and "s.v. _Paraśurāma_"; or
B. A. "s.v. yatan" and "s.v. Paraśurāma"
and would prefer either of these to the extra bother involved in encoding the reference so that italics are nicely presented.

**Comment by arlogriffiths** [comment]
I think it is best to leave control to the encoder, and so allow him to use `<foreign>` if he absolutely wants italics, while also allowing him not to bother, in which case display will be in roman typeface.

@manu, @AnneSchmiedchen, @danbalogh  : your votes please.

**Comment by danbalogh** [comment]
Fine by me so long as we explicitly permit not using `<foreign>` unless one absolutely wants to. There seems to be no TEI obstacle to using it, but perhaps it would be best to have @ajaniak 's opinion whether it would be better to use an attribute on `<citedRange>`  (e.g. `@rend="italic"`?) for this purpose.

**Comment by ajaniak** [comment]
I would offer the possibility to add the `<foreign>` rather than apply a display by default. I am not against using the attribute `@rend` to do so, even though I am not sure it fits the purpose of `@rend` in this case since we deal with display issue and not indicating how the element was presented in the source itself.

**Comment by danbalogh** [comment]
Thanks, @ajaniak . I only mentioned `@rend` because I thought an attribute may be a more proper way of doing this than an extra tag within `<citedRange>`, but I have the same reservation about it, that `@rend` is supposed to say something about the source  text, which may be a problem even though "source text" is not applicable to the bibliography. Anyway, if you say offering an optional `<foreign>` is OK, then let's go by that, since it was Arlo's original suggestion.

To summarise:
1. `<citedRange unit="entry">yatan</citedRange>` should display "s.v. yatan" without italics anywhere
2. I'll add to the EGD that optionally, if the encoder feels this is necessary, `<foreign>` may be added within `<citedRange>`, whereby the presentation would be in italics, i.e. `<citedRange unit="entry"><foreign>yatan</foreign></citedRange>` displays as "s.v. _yatan_"

We're waiting for an OK on this from @manufrancis and @AnneSchmiedchen .

**Comment by AnneSchmiedchen** [comment]
Ok from my side. Thanks.

**Comment by manufrancis** [comment]
Fine with me

**Comment by danbalogh** [comment]
EGD update done in the gdoc version (§10.4.5 and §10.3.3); I'll make the same changes in the master copy sometime soon.

**Comment by manufrancis** [comment]
Shall we close this issue?

**Comment by arlogriffiths** [comment]
all clear for me

**Comment by danbalogh** [comment]
OK to close. The change is already in the master EGD working version on GitHub.

**Comment by arlogriffiths** [comment]
This does not yet seem to have been implemented in XLST, so I am reopening the issue:

- code: ` is registered in <bibl><ptr target="bib:Zoetmulder1982_01"/><citedRange unit="entry"><foreign>devahara</foreign>, <foreign>divahara</foreign></citedRange></bibl>, with glosses`

- display: 
<img width="465" alt="Capture d’écran 2020-11-23 à 16 40 41" src="https://user-images.githubusercontent.com/835603/99982271-a58b6b80-2daa-11eb-8279-bf2f926ec319.png">

@ajaniak : can you ensure the strings tagged with `<foreign>` are displayed in italics? Or is it the occurrence of more than one term tagged with `<foreign>` that us causing the italics not to appear?
